examples/pytorch/torchscript/MNIST/MLproject

```diff
@@ -0,0 +1,20 @@
+name: mnist-torchscript
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      epochs: {type: int, default: 5}
+      batch_size: {type: int, default: 64}
+      learning_rate: {type: float, default: 1e-3}
+      tracking_uri: {type: str, default: 'http://localhost:5000'}
```

Comment:
```suggestion
```

Reply:
removed tracking uri argument and updated readme files with necessary details